PWGLF/TableProducer/Nuspex/nucleiSpectra.cxx

```diff
@@ -650,7 +682,7 @@ struct nucleiSpectra {
     nuclei::hGloTOFtracks[1]->Fill(nGloTracks[1], nTOFTracks[1]);
   }
 
-  void processData(soa::Join<aod::Collisions, aod::EvSels>::iterator const& collision, TrackCandidates const& tracks, aod::BCsWithTimestamps const&)
+  void processData(soa::Join<aod::Collisions, aod::EvSels>::iterator const& collision, TrackCandidates const& tracks, aod::BCsWithTimestamps const&, aod::McParticles const&, aod::McTrackLabels const&)
```

Comment:
Hi @ivorobye, we do not need the subscription to the mcTrackLabels and McParticles here, right?

Reply:
ah you are right, I've overlooked that leftover, fixed now!